test/rendering.spec.js

```diff
@@ -54,6 +54,18 @@ describe('Render component (DOM element) inside another', () => {
     expect(renderedComponent).toHaveClass('with-class');
     expect(renderedComponent).toBe(aComponent);
   });
+
+  it('when destroy component , then parent component should be not have rendered component', () => {
```

Comment:
Po angielskiemu to będzie bardziej tak:
`when destroy component, then parent component should not have rendered component inside` :)

Reply:
poprawię :)